dataprofiler/profilers/column_profile_compilers.py

```diff
@@ -43,20 +43,41 @@ def _create_profile(self, df_series, options=None):
         # convert all the values to string
         df_series = df_series.apply(str)
         
-        selected_columns = None
+        selected_col_profiles = None
         if options and isinstance(options, StructuredOptions):
-            selected_columns = options.enabled_columns
+            selected_col_profiles = options.enabled_columns
 
-        for column_type in self._profilers:
+        for col_profile_type in self._profilers:
             # Create profile if options allow for it or if there are no options
-            if selected_columns is None or \
-                    column_type.col_type in selected_columns:
-                column_options = None
-                if options and options.properties[column_type.col_type]:
-                    column_options = options.properties[column_type.col_type]
-                self._profiles[column_type.col_type] = \
-                    column_type(df_series.name, options=column_options)
-                self._profiles[column_type.col_type].update(df_series)
+            if selected_col_profiles is None or \
+                    col_profile_type.col_type in selected_col_profiles:
+                col_profile_options = None
+                if options and options.properties[col_profile_type.col_type]:
+                    col_profile_options = options.properties[col_profile_type.col_type]
+
+                warning_flag = False
+                try:
+                    self._profiles[col_profile_type.col_type] = \
+                        col_profile_type(df_series.name, options=col_profile_options)
+                    self._profiles[col_profile_type.col_type].update(df_series)
+                except Exception as e:
+                    import warnings
+                    warning_msg = "\n\n!!! WARNING Partial Profiler Failure !!!\n\n"
+                    warning_msg += "Profiling Type: {}".format(col_profile_type.col_type)
+                    warning_msg += "\nException: {}".format(type(e).__name__)
+                    warning_msg += "\nMessage: {}".format(e)
+
+                    # This is considered a major error
+                    if type(e).__name__ == "ValueError":
+                        raise ValueError(e)
+                    
+                    warning_msg += "\n\nFor labeler errors, try installing "
+                    warning_msg += "tensorflow and tensorflow-addons; command:\n"
+                    warning_msg += "  $ pip install tensorflow "
+                    warning_msg += "tensorflow-addons --user\n"
+
+                    warnings.warn(warning_msg, RuntimeWarning, stacklevel=2)
```

Comment:
Does this exception occur for every column of the dataset?
What doess warning_flag = False do?

Reply:
Yes, if it fails on every potential issue